yarn.lock

```diff
@@ -275,50 +297,80 @@
     rimraf "^3.0.2"
     typescript "^4.9.3"
 
-"@eclipse-glsp/dev@next":
-  version "1.1.0-next.7026c40.129"
-  resolved "https://registry.yarnpkg.com/@eclipse-glsp/dev/-/dev-1.1.0-next.7026c40.129.tgz#2ab7e736c1594e8a26556a34abbad6f43524e48c"
-  integrity sha512-VwR4xEPbEM5ZrX+WHj1iuoLfIVSmfH7iMteD/sy1AdzaL5DbytTASblVsUlO/VH6FGu8Au8EVLWVepBz7GG45Q==
+"@eclipse-glsp/dev@^1.1.0-next.acae6b0.132":
```

Comment:
When building my yarn.lock file gets updated here, could you please double check that the latest version is checked in?